org.gimp.GIMP.json

```diff
@@ -366,12 +379,37 @@
             ],
             "sources" : [
                 {
-                    "url" : "https://github.com/strukturag/libheif/releases/download/v1.9.1/libheif-1.9.1.tar.gz",
-                    "sha256" : "5f65ca2bd2510eed4e13bdca123131c64067e9dd809213d7aef4dc5e37948bca",
+                    "url" : "https://github.com/strukturag/libheif/releases/download/v1.12.0/libheif-1.12.0.tar.gz",
+                    "sha256" : "e1ac2abb354fdc8ccdca71363ebad7503ad731c84022cf460837f0839e171718",
                     "type" : "archive"
                 }
             ]
         },
+        {
+            "name": "libjxl",
+            "config-opts": [
+                "-DJPEGXL_ENABLE_PLUGINS=OFF",
+                "-DBUILD_TESTING=OFF",
+                "-DJPEGXL_WARNINGS_AS_ERRORS=OFF",
+                "-DJPEGXL_ENABLE_SJPEG=OFF",
+                "-DJPEGXL_ENABLE_BENCHMARK=OFF",
+                "-DJPEGXL_ENABLE_EXAMPLES=OFF",
+                "-DJPEGXL_ENABLE_MANPAGES=OFF",
+                "-DJPEGXL_ENABLE_SKCMS=ON",
+                "-DJPEGXL_ENABLE_JNI=OFF",
+                "-DJPEGXL_ENABLE_TCMALLOC=OFF",
+                "-DJPEGXL_ENABLE_TOOLS=OFF"
+            ],
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/libjxl/libjxl.git",
+                    "branch": "v0.6.x"
```

Comment:
please use a tag and/or a commit, not a branch. Otherwise the build is not reproducible.

Reply:
Sure, you are right. I switched to the last commit in this branch (at time of writing), because there seems to be no 0.6.x release yet (hence no tags).

@novomesk Should we really use non-released code? What about building v0.5? Well the beta branch is still a dev release, so it's up to discussion, but maybe for dependencies, it's still better to build released code.

Reply:
So replace the branch with

`commit: "20cccbf646158cb128d68db20d12f93d1fb9b6a1"` ?
https://github.com/libjxl/libjxl/commit/20cccbf646158cb128d68db20d12f93d1fb9b6a1 which is now latest in https://github.com/libjxl/libjxl/commits/v0.6.x

Reply:
> So replace the branch with

Yeah I already did the proposed changes.

Can we know how good/bad the changes are between the last release and this dev branch? I.e. is it really worth releasing an unstable dependency (even though it's dev release, or maybe even more! We already have our own unstability to not add others'! 😛)?

Reply:
Latest release of libjxl 0.5.0 is already old and have some already resolved problems.

The intention of the libjxl developers was make the release from 0.6.x branch. I am using the library in my applications and I believe it is quite good snapshot and I really prefer it more than 0.5.0

In the future we will stick to released versions.

Reply:
> What about building v0.5?

I vote against using 0.5 because:
https://github.com/libjxl/libjxl/issues/407 and https://github.com/libjxl/libjxl/issues/405

Reply:
Ok. Let's go with this because it's a dev release and we can also consider JPEG XL as a recent format hence with acceptable irregularities at the start. 🤷